OpenLIFUSonicationPlanner/OpenLIFUSonicationPlanner.py

```diff
@@ -55,6 +59,9 @@ class OpenLIFUSonicationPlannerParameterNode:
 
     """
 
+    activeVolume: vtkMRMLScalarVolumeNode
+    activeTarget: vtkMRMLMarkupsFiducialNode
```

Comment:
are these parameters getting used anywhere? i didn't see them getting used. if not then we should remove them

(i think we probably do not need to use the parameter node for this module at all)